### DISCUSSION TITLE
accelerated video, but audio at correct speed

### DISCUSSION THREAD

**Discussion by cerridwen78** [aclap-dev/video-downloadhelper#57] [Help and Q&A]
### Your question

Hello
I'm trying to download a video from a site as make a lot on times in the past, but the video I obtain is fast forward (instead the audio is ok). I don't know what is happening and why. I try to reinstall the extension and also the app, but nothing change.
An examples here: 
https://youtu.be/GMISldPrppU
Could someone help me to solve?
Thanks.

### Addon & CoApp version and details

Video DownloadHelper
Versione 8.1.4.0
Lingua del browser: it
Versione di produzione
Compilato il 2024-01-04
Opzioni di versione: browser=firefox, linuxlic=false, noyt=false

Firefox version 122.0.1 (64 bit)

Windows 10 machine

**Comment by jcfrog** [comment]
Hi,
I guess it is not the the original video but the result you get.
You should try the latest beta, it generally solves this kind of issues : https://github.com/aclap-dev/video-downloadhelper/wiki/Beta-versions
If you don't want to, the official 8.2 release should come soon.

**Reply by cerridwen78** [reply]
Thanks a lot! It solved, Beta version works well
